maryana_khoptyak/Task_1/test_get_max_value.py

```diff
@@ -0,0 +1,44 @@
+EMPTY_SEQUENCE = []
+ONLY_ONE_ELEM_NUM = ["15"]
+ONLY_ONE_ELEM_CHR = ["some  long text goes here"]
+ONLY_CHR = ["one", "two", "ten", "some  long text goes here"]
+ONLY_NUMBERS = ["1", "2", "10", "15", "-2", "16.9"]
+DATA = ["one", "1", "two", "2", "ten", "10", "15", "-2", "some  long text goes here", "16.9", "&"]
+
+
+def get_max_value(data):
+    """ Function which returns a maximum integer value from list of strings """
+    sorted_nums_list = []
+    for value in data:
+        try:
+            sorted_nums_list.append(float(value))
+        except ValueError:
+            pass
+    if sorted_nums_list:
+        return max(sorted_nums_list)
+    else:
+        None
+
+
+def test_empty_sequence():
+    assert get_max_value(EMPTY_SEQUENCE) is None, ("The is no elements in sequence, your list is", EMPTY_SEQUENCE )
```

Comment:
Is this message valid?

It looks like everything is OK when our list is empty (there are no elements in sequence).

Reply:
Removed a message as a user will never get it. Our function prevents errors when our list is empty. 
The test is still there to show that empty sequence is covered by testing data. Is it OK?

Reply:
It's true, but if for some reason our function started to return not empty list?(e.g. [[]])
It's much better to have some message here.

Reply:
Message is added there. User will be notified if anything goes wrong there.